package.json

```diff
@@ -30,22 +30,25 @@
     "ember-cli-htmlbars": "^3.0.1",
     "ember-cli-htmlbars-inline-precompile": "^2.1.0",
     "ember-cli-inject-live-reload": "2.0.1",
+    "ember-cli-sass": "^10.0.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "^1.0.0-beta.1",
     "ember-cli-uglify": "3.0.0",
     "ember-css-modules": "^1.2.1",
+    "ember-css-modules-sass": "^1.0.1",
```

Comment:
We can probably remove ember-css-modules since we will be probably be extending bootstraps native classes